docs/_documentations/mdt-eclipse-managingprojects.md

```diff
@@ -29,19 +29,21 @@ The following actions are available by right clicking on the project in the **Co
 Some actions open the default Eclipse browser. If you find that the default Eclipse browser cannot handle the content, change the default browser by navigating to **Window** > **Web Browser** and selecting a different browser from the list.
 
 - **Open Application**: Opens the application in the default Eclipse browser. This action is only available when the application is running or debugging.
-- **Open Project Overview**: Opens the overview page for this project. You can use this to see information about the project, enable or disable the project, turn auto build on and off, or edit project settings.
+- **Open Project Overview**: Opens the overview page for a project. You can use this action to see information about the project and edit project settings.
 - **Open Container Shell**: Opens a shell into your application container. This action is only available when the container is active.
 - **Open Application Monitor**: Opens the application monitor in the default Eclipse browser. Use this to monitor the activity and health of your application. This action is only available when the application is running or debugging.
+- **Open Performance Dashboard**: Opens the performance dashboard in the default Eclipse browser. This action is available only when the application is running or debugging.
 - **Import Project**: Imports your project into the Eclipse workspace.
-- **Show Log Files**: If log files are available, this action displays a list of log files. In the Eclipse **Console** view, click a log file to open it, or click **Show All** to open all available log files. The individual log files are toggle actions. Click the log file again to remove the log file, or click **Hide All** to remove all log files from the **Console** view.
+- **Disable/Enable Project**: Disables or enables the project.
+- **Show Log Files**: If log files are available, this action displays a list of log files. In the Eclipse **Console** view, click **Show All** or an individual log file toggle action to open the log file in the Eclipse **Console** view. Click the log file again to remove the log file, or click **Hide All** to remove all log files from the **Console** view.
```

Comment:
Same comment as for the readme.  Change "In the Eclipse **Console** view, click a log file to open it ..." to "Click a log file to open it ...".